examples/js/spatial-ui/spatial-utils.js

```diff
@@ -269,9 +269,7 @@ SPATIAL.utils = {
       uc = i < s || i >= 3 * s ? ur : ul;
       vc = i < 2 * s ? vh : vl;
 
-      uvs.push(uc, vc,
-              uc + ul * cosa, vc + vl * sina,
-              uc + ul * cosb, vc + vl * sinb);
+      uvs.push(uc, vc, uc + ul * cosa, vc + vl * sina, uc + ul * cosb, vc + vl * sinb);
```

Comment:
This rule makes the code less legible. It's good to be able to break a complicated list of parameters passed to a function in multiple lines

Reply:
We have the same line in one other place in this file, so I decided to do the same for this one, but I can do the other way around, put it in several lines at both places.
The indent rule with `"CallExpression": { "arguments": "first" }` allowing to align the other lines on the same position as the first argument just warned about 1 missing space indentation for this particular case.
I fixed it in the next commit, you're right that's more readable.